### PR TITLE
eMMC plugin changes

### PR DESCRIPTION
**Pull request by avri-altman-wdc**
This pull request contains few code fixes:
 - emmc: Check ffu features support: Only devices that support EXT_CSD_FFU_FEATURES should be checked for sectors done.
 - emmc: let FFU use close-ended mode - use the much-more efficient close-ended mode instead of open-ended.
 -  emmc: Add emmc quirk - Allow to use different chunk size, and add some wdc devices that needs it.

Thanks,
Avri

### CODE REVIEW

**Comment by hughsie** [comment]
@avri-altman-wdc there appear to be two precommit failures?

File: plugins/emmc/fu-emmc-device.c [hunk 1]
```diff
@@ -525,9 +541,27 @@ fu_emmc_device_write_firmware(FuDevice *device,
 	}
 	fu_progress_step_done(progress);
 
+	g_debug("programming success");
+
```
Comment (superm1) [inline plugins/emmc/fu-emmc-device.c:545]: is this really needed?  it's only shown in `--verbose --verbose`, and never changes.  I'd think the plugin progress hitting this step done is _probably_ enough.

Comment (avri-altman-wdc) [reply plugins/emmc/fu-emmc-device.c:545]: thanks.  will remove.

**Comment by superm1** [comment]
Closing in favor of v3 opened

**Pull request closed without merging**